SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -83,49 +84,26 @@ handle_zip_file() {
     if [ $? -ne 0 ]; then
         rm -f $tmp_zip_manifest
         rm -rf --preserve-root $tmp_dir
-        file_error "Error unzipping"
+        file_error_and_report_to_uploader  $BACKUP_RECIPIENT "Error unzipping file '$file'"
     fi
 
     local nc_file
     nc_file=`grep ".*.nc" $tmp_zip_manifest | head -1`
```

Comment:
If you want to match the dot before 'nc', the pattern should be `".*\.nc"` (otherwise '.' matches any character)